.travis.yml

```diff
@@ -14,6 +16,8 @@ install:
   - docker-compose run web bash -c "rake db:setup"
   - docker-compose run web bash -c "rake db:migrate"
   - docker-compose run web bower install --allow-root
+  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then gem install danger; fi'
```

Comment:
Yeah, I compared it with a string but use `= "true"` instead of `!= "false"`. Travis will return `"true"` if it's a PR, afaik. I'll confirm it later.